src/updater.ts

```diff
@@ -16,6 +16,13 @@ const REPO_CONTENTS_PATH = '/home/dependabot/dependabot-updater/repo'
 const CA_CERT_INPUT_PATH = '/usr/local/share/ca-certificates'
 const CA_CERT_FILENAME = 'dbot-ca.crt'
 
+export class UpdaterFetchError extends Error {
+  constructor(msg: string) {
+    super(msg)
+    Object.setPrototypeOf(this, UpdaterFetchError.prototype)
```

Comment:
Had to look this one up.
In TS you need to set a class as a prototype of the parent (even if you extend it) in order to use `instanceof` on an object. This is because of some `__proto__` shenanigans.

Reply:
Can confirm, this is deep magic, and I've lost my link to the changelog that describes it.